taxi/views.py

```diff
@@ -85,6 +125,24 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(DriverListView, self).get_context_data(**kwargs)
+        username = self.request.GET.get("username", "")
+        context["search_form"] = DriverSearchForm(
+            initial={"username": username}
+        )
+        context["search_form"] = DriverSearchForm()
```

Comment:
The line `context['search_form'] = DriverSearchForm()` is redundant because the search form is already initialized with initial data. Consider removing this line to avoid overwriting the form with empty initial data.